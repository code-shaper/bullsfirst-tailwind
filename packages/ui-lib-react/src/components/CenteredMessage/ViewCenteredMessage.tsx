import { ViewCenteredContainer } from '../Containers';
import { DivProps } from '../ReactProps';

export function ViewCenteredMessage({ children, ...props }: DivProps) {
  return (
    <ViewCenteredContainer className="p-6" {...props}>
      <h1 className="text-2xl text-neutral-default">{children}</h1>
    </ViewCenteredContainer>
  );
}
