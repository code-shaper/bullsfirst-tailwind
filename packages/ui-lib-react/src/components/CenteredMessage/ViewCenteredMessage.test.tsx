import { render } from '../../test/test-utils';
import { ViewCenteredMessage } from './ViewCenteredMessage';

describe('<ViewCenteredMessage />', () => {
  test('renders correctly', () => {
    const { asFragment } = render(
      <ViewCenteredMessage>ViewCenteredMessage</ViewCenteredMessage>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
