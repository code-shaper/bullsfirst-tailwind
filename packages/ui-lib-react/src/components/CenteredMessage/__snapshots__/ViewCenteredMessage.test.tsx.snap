// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewCenteredMessage /> renders correctly 1`] = `
<DocumentFragment>
  <div
    class="p-6 h-screen flex justify-center items-center text-center"
  >
    <h1
      class="text-2xl text-neutral-default"
    >
      ViewCenteredMessage
    </h1>
  </div>
</DocumentFragment>
`;
